composer.json

```diff
@@ -25,7 +25,8 @@
     },
     "require-dev": {
         "doctrine/doctrine-bundle": "^1.6.12|^2.0",
-        "doctrine/dbal": "~2.2",
+        "doctrine/dbal": "^2.13.1|^3.1",
+        "doctrine/orm": "^2.7.3",
```

Comment:
The ORM wasn't being loaded in at all, and the `AclSchemaListener` class is an ORM event listener, so I added it in here as well.

Reply:
If we add the ORM to the dependencies, we won't run tests against DBAL 3 because the ORM does not support it yet and thus blocks the installation. 🤔

Reply:
It’s one of those odd quirks right now.  A package supporting the DBAL and ORM can work on DBAL 3 support, but can’t actually get DBAL 3 installed.  Dependency chains are fun 😆